scripts/zones/Chateau_dOraguille/npcs/Halver.lua

```diff
@@ -110,7 +110,7 @@ function onTrigger(player,npc)
 		-- Mission San D'Oria 2-3 Journey Abroad 
 		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 11) then
 			player:startEvent(0x01fb);
-		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 0) then
+		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 1) then
```

Comment:
If at the very start of a mission your players have a var of 1 instead of zero, that's caused by another bug someplace else, because all missions start with zero. The gate guard should be simply adding the mission without setting a var so 1st NPC after that should be looking for zero.